### DISCUSSION TITLE
Radar as a webpage in-app (Embedded website radars)

### DISCUSSION THREAD

**Discussion by trymeouteh** [breezy-weather/breezy-weather#1178] [Ideas]
### Describe your idea

Since there are no worldwide radars and none radar APIs but a few Radar website options. I would like to suggest adding a new source called Radar.

When a user goes to the Radar page in the app, it will be a web browser and go to the radar website and view the radar on the webpage. The in-app browser can modify some of the styles to make the radar feel more native to the app.'

### Other details

Users should be informed of privacy risks when using Radar since it is not an API but a webpage in a web browser.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
This was already discussed and the conclusion was that you should use an up-to-date browser rather than rely on a webview browser that is outdated on most phones. Use a browser extension if you want to remove some styles.
